source/core/replica-set-delayed-member.txt

```diff
@@ -39,12 +39,14 @@ Delayed members:
   members. Set the priority to 0 to prevent a delayed member from
   becoming primary.
 
-- **Should be** :ref:`hidden <replica-set-hidden-members>`
+- **Must be** :ref:`hidden <replica-set-hidden-members>`
   members. Always prevent applications from seeing and querying
   delayed members.
 
 - *Do* vote in :term:`elections <election>` for primary, if :rsconf:`members[n].votes` is set to 1.
```

Comment:
I'm a little confused because here we are saying that the delayed members vote in elections for the primary, but then immediately below we say that users should ensure that they are non-voting. It seems slightly contradictory.

Do you think we should we should add a sentence to this bullet that says something like:

```
Ensuring that delayed members are non-voting by
setting :rsconf:`members[n].votes` to 0 can help improve performance.
```

Reply:
Im reading this as a warning to the user, in other words the first two bullets, which start with **Must be** are the recommendation. The *Do* is a warning if you do not address the rsconfig`members[n].votes` setting. I think the extra blurb does add value.